Controllers/FeatureController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ToggleBuddy.API.Models.DTOs.RequestDTOs;
+using ToggleBuddy.API.Models.Domain;
+using ToggleBuddy.API.Repositories.Interfaces;
+using ToggleBuddy.API.Repositories.Implementations;
+using System.Security.Claims;
+using ToggleBuddy.API.Helpers;
+using ToggleBuddy.API.Services.FeatureServices;
+
+namespace ToggleBuddy.API.Controllers
+{
+    [Route("api/{projectId:Guid}/Feature")]
+    [ApiController]
+    public class FeatureController : ControllerBase
+    {
+           private readonly IFeatureServices _featureServices;
+           private readonly IProjectRepository _projectRepository;
```

Comment:
I'm of the opinion (and I think it's a standard pattern) that Repositories should be used in services and services be used in controllers. Repositories are accessing database objects and mapping to application while services are handling application logic.

Reply:
Worked on it. You can check it out.